core/src/test/java/org/jdbi/v3/core/transaction/TestLocalTransactionHandler.java

```diff
@@ -53,4 +53,20 @@ public void testRollbackThrow() throws Exception {
             assertThat(e.getSuppressed()[0]).isSameAs(inner);
         }
     }
+
+    @Test
+    public void testThrowError() throws Exception {
+        Error error = new Error("Transaction throws!");
+
+        Mockito.when(c.getAutoCommit()).thenReturn(true);
+        Mockito.when(h.getConnection()).thenReturn(c);
+
+        try {
+            new LocalTransactionHandler().inTransaction(h, x -> {
+                throw error;
+            });
+        } catch (Throwable throwable) {
```

Comment:
could you do this with assertj's `assertThatThrownBy` instead of a regular catch, please?

Reply:
done!